### ISSUE TITLE
`sub-scale-with-window` not working

### ISSUE THREAD

**Issue by niksy** [mpv-player/mpv#1424]
When I try to set option to scale font size to window instead of video, it won’t work. Modifying `sub-scale` value does work, though.

My `mpv.conf` is this:

```
ass-force-style=FontName=Helvetica,Default.Bold=1,FontSize=20
osd-font='Helvetica:style=Bold'
osd-font-size=30
sub-scale-with-window=yes
```

Version 0.7.2 of MPV
OS X 10.10.1

**Comment by wm4** [comment]
With what kind of subtitles did you try this? E.g. paste the stream list mpv prints to the terminal on start.

**Comment by niksy** [comment]
```
Playing: video.mp4
[stream] Video (+) --vid=1 (*) (h264)
[stream] Audio (+) --aid=1 --alang=und (*) (aac)
[stream] Subs  (+) --sid=1 'video.srt' (subrip) (external)
File tags:
 major_brand: isom
 minor_version: 1
 compatible_brands: isom
 creation_time: 2015-01-05 02:17:20
AO: [coreaudio] 48000Hz stereo 2ch float
VO: [opengl] 720x404 yuv420p
AV: 00:00:06 / 00:21:22 (0%) A-V:  0.000
```

**Comment by wm4** [comment]
OK, this was confusing. I think it works a it should. I think you (and me) expected that it works like ``--osd-scale-by-window=no``, which makes the OSD always use the same pixel size, regardless of window size.

What this option does is scaling the subtitles with the window size, instead of the scaled video size. These sizes can be different if the video has a different aspect ratio than the window, and is shown with black borders. If this option is set to ``yes``, the difference due to the black border is basically ignored.

**Comment by niksy** [comment]
>  always use the same pixel size, regardless of window size.

This is what I wanted to achieve, and it’s what I thought this option does when set to `yes` (since description says "This is useful if you always want the same font size, even if the video doesn't covert the window fully").

How can that be achieved?

**Comment by wm4** [comment]
Not yet. Maybe I can try adding it as new feature.

**Comment by wm4** [comment]
Done; try ``--sub-scale-by-window=no``.

**Comment by niksy** [comment]
Is this available in 0.7.3? Setting this option in `mpv.conf` produces following error:

```sh
Error parsing option sub-scale-by-window (option not found)
```

**Comment by wm4** [comment]
No, minor releases never get new features. Just run git master if you need it now.
